packages/zoltan/src/include/zoltan.h

```diff
@@ -2802,6 +2802,27 @@ extern int Zoltan_LB_Balance(
 
 /*****************************************************************************/
 /*
+ *  Function to return the partition tree info generated by RCB.
```

Comment:
Zoltan_RCB_Partition_Tree should not be in zoltan.h, as it is not part of the public interface to Zoltan.
It can be in another include file, which would then be included in Zoltan2.